plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/aps/system/services/contentmodel/model/ContentTypeDto.java

```diff
@@ -47,28 +51,37 @@ public ContentTypeDto code(String code) {
     }
 
     @ApiModelProperty
-    public DefaultContentModel getDefaultContentModel() {
+    public String getDefaultContentModel() {
```

Comment:
Are we sure this can be done safely? Nothing else uses this call? Looks like nothing else has changed with the signature change. Seems unlikely but could be but this class is used everywhere

Reply:
I wanted @eugeniosant to confirm this, but can't get him on a call since last week.
I searched all projects and didn't find a single reference to this and the value of this property is always null.
Also, studying the UI couldn't find a possible use for this property as it is now (Enum with only 2 possible values).

My analysis shows this is safe to change, but I need someone with more experience to confirm this.

Reply:
it's very strange for me too. However, if this method is not used I wonder what it is for, and also how a user can set the default model into a content type (from the api). Is there test for this part?

Reply:
Yes, check commits on this PR for `ContentTypeResourceIntegrationTest.java `.